package.json

```diff
@@ -44,9 +44,10 @@
   "files": [
     "binding.gyp",
     "prebuilds",
+    "protos",
     "index.js",
-    "src/native_ext/*.h",
-    "src/native_ext/*.cpp",
```

Comment:
They are, but the native sources need to be included in the package, because if some combination of the prebuilt libraries doesn't exist there will be an attempt to build from source.